clients/drcachesim/tests/memref_gen.h

```diff
@@ -137,6 +146,34 @@ gen_exit(memref_tid_t tid)
     return memref;
 }
 
+inline std::vector<memref_t>
+get_memrefs_from_ir(instrlist_t *ilist, std::vector<memref_instr_t> &memref_instr_vec,
+                    const addr_t base_addr = BASE_ADDR)
```

Comment:
Do we really need this default BASE_ADDR?

Reply:
Are you suggesting we remove `base_addr` arg altogether and make it local to this function or just remove the default and have the caller always set its own `base_addr`?

Reply:
The latter.

Reply:
Done.